interface/forms/LBF/printable.php

```diff
@@ -616,6 +618,8 @@ function getContent()
 
 ?>
 
+<p style='text-align:center' class='small'>Rev. <?php echo substr($grp_last_update, 0, 10); ?></p>
```

Comment:
not sure where this variable comes from, but would be safe and surround it with a text()